test/topics_test.rb

```diff
@@ -7,6 +7,12 @@
   git_verbs = %w[GitHubbing Gitting]
   bad_github_variants = %w[Github github]
 
+  #all_related_topics = topics.reduce(Set::new) do |carry, topic|
+  #  related_topics = related_topics_for(topic)
+  #  carry.merge(related_topics)
+  #end
+  #cache_topics_exist_check!(all_related_topics)
+
```

Comment:
Is this meant to stay commented out?

Reply:
This is still used in https://github.com/github/explore/pull/4016/files#diff-de166413f9f3814bfe7482163f19a90dccc3e9aad98d11a0dd3174563d57526fR168 so I suppose, that's okay 🤔